web/README.md

```diff
@@ -2,50 +2,36 @@
 
 Label Studio is a complex, NX-managed project divided into three main components:
 
-## Main App (`apps/labelstudio`)
+## [Main App (`apps/labelstudio`)][lso]
```

Comment:
can we link from here to the README of the app/libraries?

Reply:
as I understand it leads there: https://github.com/HumanSignal/label-studio/pull/5081/files#diff-4a534cf0dab53faad7a19d1926cb05f63ebf793e827bdf11a4a19e0e654ef639R67
```
[lso]: apps/labelstudio/README.md
```